### PR TITLE
Update contact page for NFDI4Culture

### PR DESCRIPTION
**Pull request by HeyItsBATMAN**
Update contact page to reflect work on Kompakkt in the context of [NFDI4Culture](https://nfdi4culture.de/), which includes the [semantic version of Kompakkt](https://service.tib.eu/kompakkt/home) developed at [TIB](https://www.tib.eu/de/).

Screenshot:
localhost_4200_contact (1)

### CODE REVIEW

**Comment by HeyItsBATMAN** [comment]
@lozanaross @ZetOE 
Are there any missing links?
Should the GitLab version as well as the GitLab profile pages be linked?

**Comment by lozanaross** [comment]
Thanks @HeyItsBATMAN - can we make a few small changes to the OSL section:
- [ ] The link "semantic version of Kompakkt" should come here: https://gitlab.com/nfdi4culture/ta1-data-enrichment/semantic-kompakkt (ignore the 404, we'll update on our end tomorrow)
- [ ] Add link for "Open Science Lab" line - https://www.tib.eu/de/forschung-entwicklung/forschungsgruppen-und-labs/open-science
- [ ] Remove project lead / project manager lines - it makes it too wordy.
- [ ] Rename "Project Team" to "Semantic Kompakkt Team"
- [ ] Update the team in this order (with GitLab links, not GitHub):
Ina Blümel [Email](mailto:Ina.Bluemel@tib.eu)
Paul Duchesne  [GitLab](https://gitlab.com/paulduchesne)
Lukas Günther  [GitLab](https://gitlab.com/luguenth1)
Lozana Rossenova [GitLab](https://gitlab.com/lozanaross)
Zoe Schubert  [GitLab](https://gitlab.com/zetoheh)
Lucia Sohmen [GitLab](https://gitlab.com/sohmenl)

@luguenth - can we make identical changes to our version (and add the separator line between the projects - it looks good).

**Comment by HeyItsBATMAN** [comment]
@lozanaross I updated all of your requests in the last commit.

Please tell me once you confirm that everything's correct and the link to the semantic version is publicly available.

Screenshot:
Screen Shot 2022-11-09 at 20 42 35

**Review by lozanaross — APPROVED** [review]
All looks good, approved to go live.
